.env.sample

```diff
@@ -12,4 +17,6 @@ CONNECTOR_EXPORT_FILE_TXT_ID=ca715d9c-bd64-4351-91db-33a8d728a58b
 CONNECTOR_IMPORT_FILE_STIX_ID=72327164-0b35-482b-b5d6-a5a3f76b845f
 CONNECTOR_IMPORT_DOCUMENT_ID=c3970f8a-ce4b-4497-a381-20b7256f56f0
 SMTP_HOSTNAME=localhost
-ELASTIC_MEMORY_SIZE=4G
\ No newline at end of file
+
+ELASTIC_MEMORY_SIZE=4G
+ELASTICSEARCG_URL=""
```

Comment:
These env vars connect to Elasticsearch and Disarm Framework:
```
ELASTIC_USER=
ELASTICPASSWORD=
ELASTIC_IP_1=
ELASTIC_IP_2=
ELASTIC_IP_3=
ELASTIC_PORT=
DISARM_CONNECTOR_ID=815330ab-ec11-4388-a30b-62a213cdbebb
```

Reply:
> These env vars connect to Elasticsearch and Disarm Framework:
> 
> ```
> ELASTIC_USER=
> ELASTICPASSWORD=
> ELASTIC_IP_1=
> ELASTIC_IP_2=
> ELASTIC_IP_3=
> ELASTIC_PORT=
> DISARM_CONNECTOR_ID=815330ab-ec11-4388-a30b-62a213cdbebb
> ```

@VinneyJ I think, this should be introduced in #3 where they're implemented

Reply:
Sure let me add it in the env.sample. @thepsalmist